service/lib/agama/dbus/manager.rb

```diff
@@ -114,6 +124,11 @@ def current_installation_phase
         return INSTALL_PHASE if backend.installation_phase.install?
       end
 
+      # States whether installation runs on iguana
+      def iguana_backend
+        backend.iguana? ? IGUANA_USED : IGUANA_NOT_USED
```

Comment:
NP: I think these constant are not needed

```suggestion
        backend.iguana?
```

Reply:
you're right .. it's a leftover from previous implementation